input_parameters.conf

```diff
@@ -21,6 +21,12 @@ savenpy:       0
 # Optional save of incremental time series products (TIMESERIES/MERGE)
 savetsincr:    0
 
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Integer parameters
+
+# Number of sigma to report velocity error. 1 or 2. Default: 2 (TIMESERIES/MERGE)
```

Comment:
I think this parameter is only relevant to TIMESERIES step, since it is applied in `pyrate/core/timeseries.py`. Delete "MERGE" in this comment